### PR TITLE
chore: set up ci pipelines

### PR DESCRIPTION
**Pull request by peter-vizi-pager**
I'm setting up the standard ci pipeline and removing circle-ci from here.

### CODE REVIEW

File: cloudbuild.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    entrypoint: npm
+    args: ['i', '--quiet', '--package-lock-only']
+
+  - id: npm-audit
+    name: 'gcr.io/$PROJECT_ID/cloudbuilders/audit-filter:0.5.0'
+    secretEnv: ['NPM_TOKEN']
+    entrypoint: "/bin/bash"
+    args:
+      - -c
+      - npm audit --production --json | audit-filter
```
Comment (BlaineEhrhart) [inline cloudbuild.yaml:14]: Isn't this the "old" way of doing audit filtering?

Comment (BlaineEhrhart) [reply cloudbuild.yaml:14]: https://github.com/pagerinc/plugins-service/blob/master/cloudbuild.yaml#L18-L28

I seem to recall having to come up with a different way of doing it but I don't think I documented the reasoning behind it. If this works I won't slow you down. I'll try to do some research as to why I had to do this.

**Review by BlaineEhrhart — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "description": "Hapi Prometheus Plugin and optional endpoint",
   "main": "lib/index.js",
   "scripts": {
-    "test": "lab -La @hapi/code -t 100",
+    "test": "lab",
```
Comment (markguyver) [inline package.json:7]: The `-t 100` sets the test coverage threshold.  So you are removing the requirement for 100% code coverage...

Comment (peter-vizi-pager) [reply package.json:7]: It's in the .labrc.js: threshold = 100

File: cloudbuild.yaml (hunk 1 above)
Comment (peter-vizi-pager) [reply cloudbuild.yaml:14]: I've only seen audit-filter and audit-exclude before, not sure why better-npm-audit is better :D

We should probably settle on one as part of the coding standards/etc. discussion.

Comment (BlaineEhrhart) [reply cloudbuild.yaml:14]: Agreed

**Pull request merged**